src/views/Settings.vue

```diff
@@ -116,24 +113,23 @@
               <b-button variant="danger" @click="rebootPrompt">Reboot</b-button>
               <b-modal
                 ref="reboot-modal"
-                title="Are you sure?"
+                :title="hasRebooted ? 'Successfully rebooted' : 'Are you sure?'"
                 no-close-on-backdrop
                 no-close-on-esc
-                @ok="reboot($event)"
+                @ok.prevent="reboot()"
                 :cancel-disabled="isRebooting || hasRebooted"
-                :ok-disabled="isRebooting">
+                :ok-disabled="isRebooting"
```

Comment:
This no longer works properly, ok isn't clickable once reboot is complete 🤔.

Reply:
I'm not sure why since it looks like you update it here: https://github.com/lukechilds/umbrel-dashboard/pull/1/files#diff-ba7c5b72f7d70f958bd54775c7f6b091R114

Maybe I'm misunderstanding how the data follows through?

Reply:
Ahh, fixed it, it was `ok.prevent` that was causing the issue.

We still need to call the handler, we just need to pass `event` through so we can `preventDefault()` on reboot.

```
diff --git a/src/views/Settings.vue b/src/views/Settings.vue
index fcc22db..c41ee07 100644
--- a/src/views/Settings.vue
+++ b/src/views/Settings.vue
@@ -116,7 +116,7 @@
                 :title="hasRebooted ? 'Successfully rebooted' : 'Are you sure?'"
                 no-close-on-backdrop
                 no-close-on-esc
-                @ok.prevent="reboot()"
+                @ok="reboot($event)"
                 :cancel-disabled="isRebooting || hasRebooted"
                 :ok-disabled="isRebooting"
               >
@@ -269,10 +269,11 @@ export default {
       this.$store.commit("system/setHasRebooted", false);
       this.$refs["reboot-modal"].show();
     },
-    async reboot() {
+    async reboot(event) {
       if (this.hasRebooted) {
         return;
       }
+      event.preventDefault();
       await this.$store.dispatch("system/reboot");
     }
   },
```

Reply:
Strange, 'cause `ok.prevent` and `event.preventDefault` are supposed to be equivalent AFAIU. 
[[Event Modifiers](https://vuejs.org/v2/guide/events.html#Event-Modifiers)]